src/modules/form/components/DynamicField.tsx

```diff
@@ -144,6 +144,7 @@ const DynamicField: React.FC<DynamicFieldProps> = ({
 
   const commonInputProps: DynamicInputCommonProps = {
     label,
+    ariaLabel: item.text || undefined,
```

Comment:
we need to holistically audit how aria labeling is working for all the input components, because I am sure there are other places that are missed

Reply:
I am hoping storybook can help.